ldap/README.md

```diff
@@ -29,9 +29,11 @@ It creates 3 Organisational Units (ou):
 
 The basic users:
  * ```testuser``` has the USER role. The password is **testuser**.
+ * ```testpendinguser``` has the PENDING role, which means an admin has to validate it. The password is **testpendinguser**.
  * ```testreviewer``` has the USER & GN_REVIEWER roles. The password is **testreviewer**.
  * ```testeditor``` has the USER & GN_EDITOR roles. The password is **testeditor**.
  * ```testadmin``` has the USER, GN_ADMIN, ADMINISTRATOR and MOD_* roles. The password is **testadmin**
+ * ```testdelegatedadmin``` has the USER & ORGADMIN roles. The password is **testdelegatedadmin**
```

Comment:
has the `USER` role. Is able to grant the `EXTRACTORAPP` & `GN_EDITOR` roles to members of the `psc` & `c2c` orgs.